http_post.go

```diff
@@ -132,7 +134,9 @@ func (n *HTTPPostNode) postRow(row *models.Row) {
 		return
 	}
 
-	req.Header.Set("Content-Type", "application/json")
+	if contentType != "" {
```

Comment:
Should `contentType` be settable from the config?

Reply:
It is already via the headers system.

Reply:
Ah good point.